### ISSUE TITLE
the helloworld does not work

### ISSUE THREAD

**Issue by yota-code** [cannatag/ldap3#893]
The initial helloworld contains a typo... and a bug:

```
from ldap3 import Server, Connection, SAFE_SYNC
server = Server('my_server')
conn = Connection(s, 'my_user', 'my_password', strategy=SAFE_SYNC, auto_bind=True)
status, result, response, _ = conn.search('o=test', '(objectclass=*)')  # usually you don't need the original request (4th element of the return tuple)
```

when `s` is changed to `server`, I still get:

```
> python3 auth.py
Traceback (most recent call last):
  File "auth.py", line 5, in <module>
    conn = Connection(server, 'my_user', 'my_password', strategy=SAFE_SYNC, auto_bind=True)
TypeError: __init__() got an unexpected keyword argument 'strategy'
````

my config:

```
> python3
Python 3.8.6 (default, Oct 14 2020, 17:24:01)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ldap3
>>> ldap3.__version__
'2.8.1'
```

**Comment by cannatag** [comment]
Hi, at https://ldap3.readthedocs.io/en/latest/index.html there is the updated version of the code. Can you check it?

Thanks,
Giovanni

**Comment by cannatag** [comment]
Fixed in ldap3 v2.9
